src/Domain/Entities/ApplicationTask.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    public class ApplicationTask
```

Comment:
@GasperPaul In case I name class "Task" it will be confused with Thread.Task class. As the result, we will be forced to explicitly specify namespace each time before using Task class. Thus, I've decided to change the name to ApplicationTask.